Src/libCZI/libCZI.h

```diff
@@ -82,11 +82,14 @@ namespace libCZI
 		std::string repositoryTag;
 	};
 
-	/// Gets the version of the library.
+	/// Gets the version of the library. For versioning libCZI, SemVer2 (<https://semver.org/>) is used.
+    /// Note that the value of the tweak version number does not have a meaning (as far as SemVer2 is concerned).
```

Comment:
I have to confess - the current state of "formatting" is a bit of a mess, it has been edited with different editor-configurations and tab-vs-spaces is not consistent across the codebase.
Since [EditorConfig ](https://editorconfig.org/) is widely supported nowadays (see also [here](https://learn.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2022)), I'd reckon I'd opt for a new task - add "EditorConfig"-configuration to project, and tidy up the formatting in all files.

Reply:
I fixed the formatting manually for this line, hopefully the last time this is necessary :-)

Added an issue -> #31